tests/integration/test_db.py

```diff
@@ -196,85 +194,6 @@ async def test_relation_data_is_updated_correctly_when_scaling(ops_test: OpsTest
                 psycopg2.connect(primary_connection_string)
 
 
-@markers.juju2
-@pytest.mark.group(1)
-@markers.amd64_only  # sentry snap not available for arm64
-async def test_sentry_db_blocked(ops_test: OpsTest, charm: str) -> None:
```

Comment:
Test fails consistently with:
```
juju.errors.JujuAPIError: cannot add relation "haproxy:reverseproxy sentry1:http": unknown series for version: "24.04"
```
Since the sentry charm doesn't seem to be maintained I propose removing the test.